### PR TITLE
[ENG-3788]Delete folder

### PR DESCRIPTION
**Pull request by adlius**
-   Ticket: []
-   Feature flag: n/a

### CODE REVIEW

**Review by brianjgeiger — APPROVED** [review]

**Review by brianjgeiger — CHANGES_REQUESTED** [review]
Actually, on [lib/osf-components/addon/components/file-browser/folder-item/template.hbs](https://github.com/CenterForOpenScience/ember-osf-web/pull/1493/files#diff-29f8f685b1ac1464606b184c1d2825149c0ceed5bfdfed3f98afd91ed581c504) Line 39, the folder menu is not being shown if you're on a provider where download isn't allowed, because it was the only option in the menu at the time. This logic will need to be fixed so that it will show if there are any available options in the folder menu.

**Review by brianjgeiger — APPROVED** [review]

File: lib/osf-components/addon/components/file-browser/folder-item/component.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+export default class FileBrowser extends Component {
```
Comment (futa-ikeda) [inline lib/osf-components/addon/components/file-browser/folder-item/component.ts:4]: copy-pasta?

File: lib/osf-components/addon/components/file-browser/folder-item/styles.scss [hunk 2]
```diff
@@ -41,12 +41,17 @@
     background-image: linear-gradient(90deg, rgba(236, 240, 241, 0) 0%, #ecf0f1 12%, #ecf0f1 89%, rgba(236, 240, 241, 0) 100%);
 }
 
-.FileList__item__options {
+.FileList__item__name {
```
Comment (futa-ikeda) [inline lib/osf-components/addon/components/file-browser/folder-item/styles.scss:44]: I was just starting to look into this, so thank you for fixing this!

**Review by futa-ikeda — APPROVED** [review]
Looks good! Just one copy-pasta in the folder-item/component.ts file, but I am adding that component file in my PR too, so I can update that when I merge this

**Pull request merged**